tests/app/dao/test_notification_dao.py

```diff
@@ -1201,7 +1204,9 @@ def test_should_exclude_test_key_notifications_by_default(
 
 def test_get_financial_year():
     start, end = get_financial_year(2000)
-    assert start.tzinfo == pytz.utc
-    assert start.isoformat() == '2000-04-01T00:01:00+00:00'
-    assert end.tzinfo == pytz.utc
-    assert end.isoformat() == '2001-04-01T00:01:00+00:00'
+    assert str(start) == '2000-03-31 23:00:00'
+    assert str(end) == '2001-03-31 23:00:00'
+
+
+def test_get_april_fools():
+    assert str(get_april_fools(2016)) == '2016-03-31 23:00:00'
```

Comment:
can you also assert that ` assert get_april_fools(2016).tzinfo is None`? you can just do it in the same test if you want